guardian/src/main/java/com/auth0/android/guardian/sdk/ParcelableNotification.java

```diff
@@ -115,6 +119,7 @@ public static ParcelableNotification parse(@NonNull Bundle pushNotificationPaylo
         String hostname = pushNotificationPayload.getString(HOSTNAME_KEY);
         String enrollmentId = pushNotificationPayload.getString(ENROLLMENT_ID_KEY);
         String transactionToken = pushNotificationPayload.getString(TRANSACTION_TOKEN_KEY);
+        String transactionLinkingId = pushNotificationPayload.getString(TRANSACTION_LINKING_ID_KEY);
```

Comment:
Consider using pushNotificationPayload.containsKey(TRANSACTION_LINKING_ID_KEY) first, as the transaction linking id is optional.

I am not sure how android.os.Bundle works if the key is not found.

Reply:
nevermind, it looks like it returns null in this case
https://developer.android.com/reference/android/os/BaseBundle#getString(java.lang.String)